src/openapi-generator.ts

```diff
@@ -974,6 +974,10 @@ export class OpenAPIGenerator {
       };
     }
 
+    if (isZodType(zodSchema, 'ZodPipeline')) {
+      return this.toOpenAPISchema(zodSchema._def.in, isNullable, defaultValue);
```

Comment:
Well this was easy 😄

Reply:
Are you sure that `in` is the right thing to use here?

I would expect something like this:
- Use `in` for `parameters` as this is the possible input, from which an instance can be created
- Use `out` for `schemas` as this is the resulting type that will come out